.travis.yml

```diff
@@ -15,20 +15,11 @@ notifications:
 before_script:
   - export PATH=$HOME/.local/bin:$PATH
 
-branches:
-  only:
-    - master
-    - staging
-    - trying
-    - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
```

Comment:
keep these?

Reply:
If we keep these, that means there will be no CI run on pull requests.

Reply:
No.

Reply:
I think these are only for branch pushes, not PRs.